tools/src/scie_pants/install_pants.py

```diff
@@ -105,43 +73,48 @@ def install_pants_from_pex(
         try:
             ptex.fetch_to_fp(pex_url, pants_pex.file)
         except subprocess.CalledProcessError as e:
-            #  if there's only one dot in version, specifically suggest adding the `.patch`)
-            suggestion = (
-                "Pants version format not recognized. Please add `.<patch_version>` to the end of the version. For example: `2.18` -> `2.18.0`.\n\n"
-                if version.base_version.count(".") < 2
-                else ""
-            )
             fatal(
-                f"Wasn't able to fetch the Pants PEX at {pex_url}.\n\n{suggestion}"
```

Comment:
The suggestion has moved to the configuration binding, as an unknown version will fail at that point and never get here.